### PR TITLE
Add docs for daml script --ide-ledger

### PR DESCRIPTION
**Pull request by samuel-williams-da**
Required for when https://github.com/digital-asset/daml/pull/16547 is merged

### CODE REVIEW

**Comment by basvangijzel-DA** [comment]
Added daml-docs team for their input, looks good from my side (given comment).

File: docs/2.7.0/docs/daml-script/index.rst [hunk 1]
```diff
@@ -320,6 +320,21 @@ translated to Daml script but there are a few things to keep in mind:
 
 .. _daml-script-distributed:
 
+Use Daml Script with the IDE Ledger
+===================================
+
+Similarly to running ``daml test`` or the codelens in VSCode, you can use
+``daml script`` to run the scripts in a given dar file within the IDE Ledger,
```
Comment (loiskilberg-da) [inline docs/2.7.0/docs/daml-script/index.rst:327]:
```suggestion
``daml script`` to run the scripts in a given DAR file within the IDE Ledger.
```

File: docs/2.7.0/docs/daml-script/index.rst [hunk 2]
```diff
@@ -320,6 +320,21 @@ translated to Daml script but there are a few things to keep in mind:
...
+a fully in-memory child process of ``daml script``, allowing you to quickly
```
Comment (loiskilberg-da) [inline docs/2.7.0/docs/daml-script/index.rst:328]:
```suggestion
This is a fully in-memory child process of ``daml script``, allowing you to quickly
```

File: docs/2.7.0/docs/daml-script/index.rst [hunk 3]
```diff
@@ -320,6 +320,21 @@ translated to Daml script but there are a few things to keep in mind:
...
+invoke a script without having to spin up a ledger in the background.
+
+To run ``daml script`` in this mode, you should provide the ``--ide-ledger`` flag.
+This flag is not compatible with ``--ledger-host``, ``--ledger-port``,
+``--participant-config`` (as you'll see in the next section), and ``--json-api``.
```
Comment (loiskilberg-da) [inline docs/2.7.0/docs/daml-script/index.rst:333]:
```suggestion
``--participant-config`` (described more in the next section), and ``--json-api``.
```

File: docs/2.7.0/docs/daml-script/index.rst [hunk 4]
```diff
@@ -320,6 +320,21 @@ translated to Daml script but there are a few things to keep in mind:
...
+Note that since this uses an in-memory ledger, no state will be preserved once
+the script finished, you'll only receive a success flag and optionally the script result
```
Comment (loiskilberg-da) [inline docs/2.7.0/docs/daml-script/index.rst:335]:
```suggestion
the script finishs. You will only receive a success flag and, optionally, the script result
```

File: docs/2.7.0/docs/daml-script/index.rst [hunk 5]
```diff
@@ -320,6 +320,21 @@ translated to Daml script but there are a few things to keep in mind:
...
+if using ``--output-file``.
```
Comment (loiskilberg-da) [inline docs/2.7.0/docs/daml-script/index.rst:336]:
```suggestion
if you use ``--output-file``.
```

**Review by loiskilberg-da — COMMENTED** [review]
just made some language suggestions

**Pull request merged**